docs/music.rst

```diff
@@ -120,11 +115,11 @@ Functions
     blocking call is interrupted or, in the case of a background call, a new
     frequency is set or ``stop`` is called (see below).
 
-.. py:function:: stop(pin=pin0)
-                 stop(pin=(pin_speaker, pin0))
+.. py:function:: stop(pin=pin1)
     
-    Stops all music playback on a given pin, eg. ``music.stop(pin_speaker)``
-    **V2**. If no pin is given, eg. ``music.stop()`` pin0 is assumed.
+    Stops all music playback on the built-in speaker and an optional pin, eg.
```

Comment:
Instead of "optional" could could say something like "a pin outputting sound"?

Reply:
UPdated in https://github.com/bbcmicrobit/micropython/pull/695/commits/05c8b169faa12a8b27fa760a3f374dd47bc999a1